### PR TITLE
Create a new post is now working!

### PR DESCRIPTION
**Pull request by tissyane**
Implemented router, repository and controller to insert and create a new post. 

Also, created a generic middleware to validate all schemas in the application.

### CODE REVIEW

File: src/controllers/post.controller.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import { insertNewPost } from "../repositories/post.repository.js";
+
+const createPost = async (req, res) => {
+  // TODO:Pegar o userId da validação(middleware) e corrigir no repository
+  //const userId = res.locals;
```
Comment (SauloDriven) [inline src/controllers/post.controller.js]: please remove comments

File: src/controllers/post.controller.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  const { url, description } = req.body;
+
+  try {
+    await insertNewPost(url, description);
+
+    return res.sendStatus(200);
+  } catch (error) {
+    console.log(error.message);
```
Comment (SauloDriven) [inline src/controllers/post.controller.js]: remove console

**Review by guilhermeicarofr — APPROVED** [review]
Approved for now, we still have to add the entire logic to identify tags on the post description and check if they already exist, insert tags on database and relate posts to tags on the postsHashtags database table. But for a basic feature all seems OK

**Review by SauloDriven — APPROVED** [review]
please apply the suggestions before merging

**Pull request merged**